packages/preset-dark/src/index.ts

```diff
@@ -107,7 +107,7 @@ export default {
       my: 4,
       borderCollapse: 'separate',
       borderSpacing: 0,
-      [['th', 'td']]: {
+      [['th', 'td'].toString()]: {
```

Comment:
Did you decide not to go this route here?
```suggestion
      'th, td': {
```

Reply:
Okay boss I did it properly